src/rust/engine/nailgun_process_map/nailgun_process_manager/src/main.rs

```diff
@@ -0,0 +1,55 @@
+
+use sysinfo::{SystemExt};
+use std::collections::btree_map::BTreeMap;
+
+use process_execution::{ExecuteProcessRequest, Platform};
+use nailgun_process_map::NailgunProcessMap;
+use std::time::Duration;
+use std::collections::btree_set::BTreeSet;
+use std::io::Read;
+
+fn main() {
+    let tool_name = String::from("zinc");
+    let startup_options = ExecuteProcessRequest {
+        argv: vec![
+            String::from("/Library/Java/JavaVirtualMachines/TwitterJDK/Contents/Home/bin/java"),
```

Comment:
can
```
vec!["x", "y"].into_iter().map(|x| x.to_string).collect()
```
rather than wrapping all of these calls.

Reply:
Thanks! The next time I need to change that CLI I'll transform this